File: app/src/main/java/com/rr/rhythms/Business/Graphs/Graph33.java
package com.rr.rhythms.Business.Graphs;

import android.graphics.Color;

/**
 * Created by Huruk on 7/5/2016.
 */
public class Graph33 extends GraphBase {
    public static final int PERIOD = 33;

    @Override
    public int GetPeriod() {
        return PERIOD;
    }

    @Override
    public int getColor() {
        return Color.GREEN;
    }

    @Override
    public String getName() {
        return Integer.toString(PERIOD);
    }
}

File: app/src/main/java/com/rr/rhythms/Business/Graphs/Graph33.kt
package com.rr.rhythms.Business.Graphs

import android.graphics.Color

/**
 * Created by Huruk on 7/5/2016.
 */
class Graph33 : GraphBase() {
    override val color: Int = Color.GREEN

    override val name: String = period.toString()

    override val period: Int
        get() = 33
}
